VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -137,8 +135,8 @@ public void updateDrawState(TextPaint ds) {
                         context.getResources(), finalStringRes, valuesMap));
                 // If time is not available, then reduce the whitespaces
                 // surrounding it's placeholder in the template to one.
-                if (TextUtils.isEmpty(formattedText) || TextUtils.isEmpty(authorLabel)) {
-                    formattedText = removeDuplicateWhitespace(formattedText);
+                if (TextUtils.isEmpty(formattedTime)) {
+                    formattedText = removeDuplicateWhitespaces(formattedText);
```

Comment:
I'm not really sure this is the right way to do this. Who knows how things are going to be combined in different languages. It might not just be a spacing thing. Maybe better to have different template strings?